workflows/prognostic_c48_run/tests/test_nudging.py

```diff
@@ -190,6 +190,24 @@ def test_get_nudging_tendency(
         assert result[name].attrs["units"] == tendency.attrs["units"]
 
 
+def test_nudging_tendency_fillna(
```

Comment:
This is a lot of test code for the small change. It's a tricky integration to test, and the test is quite coupled to the implementation (e.g. sharing STATE_NAME_TO_TENDENCY). The test is also 100x longer than the added code. All this suggests the API/behavior being tested could be problematic. 

Maybe it would be easier test if we  always fillna when *applying* any kind of tendency? This would also have the benefit of working in other settings than the nudging. I'm not sure if there is a convenient `apply_tendency` function. If so, you should refactor the fillna there. Otherwise, I suggest just removing  this test.

Reply:
Moving to the `add_tendency` function as discussed.